pos_sequence_ref_number/models/pos_order.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 Acsone SA/NV (http://www.acsone.eu) and
+# Eficent Business and IT Consulting Services S.L. (http://www.eficent.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import api, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    @api.model
+    def sequence_number_sync(self, vals):
+        next_number = vals.get('sequence_ref_number', False)
+        next_number = int(next_number) if next_number else False
+        if vals.get('session_id') and next_number is not False:
+            session = self.env['pos.session'].sudo().browse(vals['session_id'])
+            if next_number != session.config_id.sequence_id.number_next_actual:
+                session.config_id.sequence_id.number_next_actual = next_number
+        if vals.get('sequence_ref_number') is not None:
```

Comment:
I think we should store the ticket number on the order. We have an information which could be usefull, so why not keeping it.

Reply:
@benwillig I don't think this could be useful, here sequence_ref_number is only used in the case where sequence of the UI is different from the next generated by the system.

Reply:
@benwillig Could you update your review ?